README.md

```diff
@@ -1,260 +1,64 @@
-
-You can then run the following command in the root of this git repository:
-```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$BUNDLE_TAG -d deploy/olm-catalog/windows-machine-config-operator/manifests \
---channels alpha --default-channel alpha --image-builder podman
-```
-The variables in the command should be changed to match the container image repository you wish to store the bundle in.
-You can also change the channels based on the release status of the operator.
-This command should create a new bundle image. Bundle image and operator image are two different images. 
+* OKD/OCP 4.6 cluster running on Azure or AWS, configured with hybrid OVN Kubernetes networking
 
-You should then push the newly created bundle image to the remote repository:
-```shell script
-podman push $BUNDLE_REPOSITORY:$BUNDLE_TAG
+## Usage
+Once the `openshift-windows-machine-config-operator` namespace has been created, a secret must be created containing
+the private key that will be used to access the Windows VMs. The private key should be in PEM encoded RSA format.
 ```
-
-You should verify that the new bundle is valid:
-```shell script
-operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_TAG --image-builder podman
-```
-
-### Creating a new operator index
-You can skip this step if you want to run the operator for [developer testing purposes only](#development-workflow)
-
-An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
-cluster.
-
-```shell script
-opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$INDEX_TAG --container-tool podman
+# Change path as necessary
+oc create secret generic cloud-private-key --from-file=private-key.pem=/path/to/key.pem -n openshift-windows-machine-config-operator
 ```
 
-You should then push the newly created index image to the remote repository:
-```shell script
-podman push $INDEX_REPOSITORY:$INDEX_TAG
-```
-
-#### Editing an existing operator index
-An existing operator index can have bundles added to it:
-```shell script
-opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
-```
-and removed from it:
-```shell script
-opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
-```
+Below is the example of an Azure Windows MachineSet which can create Windows Machines that the WMCO can react upon.
+Please note that the windows-user-data secret will be created by the WMCO lazily when it is configuring the first
+Windows Machine. After that, the windows-user-data will be available for the subsequent MachineSets to be consumed.
+It might take around 10 minutes for the Windows VM to be configured so that it joins the cluster. Please note that
+the MachineSet should have following labels:
+* `machine.openshift.io/os-id: Windows`
+* `machine.openshift.io/cluster-api-machine-role: worker`
+* `machine.openshift.io/cluster-api-machine-type: worker`
```

Comment:
I suggest using italics here. We should reserve `blah` for code and yaml blocks. There is too much `blah` in the README now so it is for aesthetic reasons :-)